.github/workflows/.env/nightly-tests/max_versions.env

```diff
@@ -4,4 +4,4 @@ kind_version='v0.24.0'
 helm_version='v3.14.4'
 argocd_version='v2.8.4'
 istio_version='1.22.0'
-k8sgateway_api_version='v1.2.0'
+k8sgateway_api_version='v1.2.1'
```

Comment:
Why don't we have the equivalent for https://github.com/solo-io/gloo/blob/main/.github/workflows/.env/pr-tests/versions.env? 

I know this is a bit outside the scope of this PR (and I'm happy if we do it in a separate PR), but given that we identified this drift, could we add the k8sgateway_api_version to the pr-test env file?

Reply:
@sam-heilbron ^ is being resolved by https://github.com/solo-io/gloo/pull/10390.